src/tour/tour.service.ts

```diff
@@ -56,9 +63,7 @@ export class TourService {
     const deleteResult = await this.tourRepository.delete({ id, user });
 
     if (deleteResult.affected === 0) {
```

Comment:
What if something else goes wrong (e.g. no db connection or no cascading delete)? I assume deleteResult.affecteded will also return 0 in this case?